### PR TITLE
Dem 222 Failing tests

### PR DESCRIPTION
**Pull request by indytechcook**
Fixing some tests and jshint issues

@tladendo @vietnogi

### CODE REVIEW

**Comment by vietnogi** [comment]
Can you also add the Jira title to the PR so we don't have to look it up.

File: src/client/app/feature/feature.service.js [hunk 1]
```diff
@@ -8,16 +8,70 @@
...
-      {id: 'social-login', name: "Social Login", explanation: "", description: "Login with Twitter, Facebook, LinkedIn, or Google", selected: false},
-      {id: 'profiles', name: "Profiles", explanation: "", description: "People can enter information about themselves", selected: false},
-      {id: 'accept-payments', name: "Accept Payments", explanation: "", description: "Accept credit cards, PayPal, Bitcoin", selected: false},
-      {id: 'ratings', name: "Ratings/Reviews", explanation: "", description: "People leave reviews and/or rate things", selected: false},
-      {id: 'location', name: "Location-based or Navigation Element", explanation: "", description: "Location-based or Navigation Element", selected: false},
-      {id: 'sharing', name: "Sharing Functions", explanation: "", description: "Share on Twitter, Facebook, Email", selected: false},
-      {id: 'api', name: "An API", explanation: "", description: "Data can be connected with a website or other", selected: false},
-      {id: 'search', name: "Search", explanation: "", description: "People can search/browse/filter", selected: false}
+    var features = [
```
Comment (vietnogi) [inline src/client/app/feature/feature.service.js]: much cleaner, thnx

**Comment by tladendo** [comment]
lgtm

File: src/client/app/feature/feature.service.js (hunk 1 above)
Comment (indytechcook) [inline src/client/app/feature/feature.service.js]: I use webstorm, there is a plugin for everything :) 

Call me crazy, I actually don't like the look of aligned : and = and such.  Though if the team decides to which then I'll adjust.

**Pull request merged**